src/behapy/fp.py

```diff
@@ -3,8 +3,6 @@
 import scipy.signal as sig
 from collections import namedtuple
 from typing import Iterable
-
```

Comment:
Why the deleted whitespace?

Reply:
ach, i don't recall why i changed that -- pls ignore!